### DISCUSSION TITLE
QR Code Frame not visible

### DISCUSSION THREAD

**Discussion by mariomurrent-softwaresolutions** [capawesome-team/capacitor-mlkit#154] [Q&A] [answered]
In the tutorial there is the nice white frame for the code. (https://capawesome.io/plugins/mlkit/barcode-scanning/)
 Is this something which i manually need to add? I indeed see the camera and scanning works, but I don't see the nice white frame.

**Comment by robingenz** [comment, marked as answer]
Yes, you need to add this yourself. Feel free to copy the code form our [official demo app](https://github.com/robingenz/capacitor-mlkit-plugin-demo).
